src/components/cc-logs/cc-logs.test.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-env node, mocha */
+
+import { testAccessibility } from '../../../test/helpers/accessibility.js';
+import { getStories } from '../../../test/helpers/get-stories.js';
+import * as storiesModule from './cc-logs.stories.js';
+
+const storiesToTest = getStories(storiesModule);
+
+describe(`Component: ${storiesModule.default.component}`, function () {
+  testAccessibility(storiesToTest);
```

Comment:
**nitpick:** I wonder if we should ignore some of the stories like `dataLoadedWithFollow` (the test will probably be done before any message is added anyway).

Reply:
I would be ok to ignore some stories, this is a great idea. But I don't want to introduce here a story filtering by name. 

I would prefer to add a property like `skipTest` (`false` by default) and modify the `getStories()` function to filter on that property.

What do you think?

Reply:
see #731